.snyk

```diff
@@ -7,4 +7,34 @@ ignore:
         reason: >-
           No remediation available yet; Not affecting us since the storage is not accessible to any other client
         expires: 2021-01-10T06:00:00.000Z
-patch: {}
\ No newline at end of file
+  SNYK-PYTHON-JINJA2-455616:
+    - '*':
+        reason: >-
+          Must wait until CKAN2.8 ready for deployment
```

Comment:
Similar comment as [inventory-app](https://github.com/GSA/inventory-app/pull/51#discussion_r454014385). We have the "why we're not upgrading" but need to include "why the risk is acceptable". Also, any High severity should have a shorter expiration (30 days).

Reply:
@adborden, let me know if these changes are sufficient. Prepping all notes to go into documentation for O&M role...